freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/kerberosmgmt/doc/KeytabModelNotes.java

```diff
@@ -7,8 +7,9 @@
             + "the response is base64 encoded.";
     public static final String GET_KEYTAB_NOTES = "Retrieves the existing keytab for the service principal derived from the host and service provided. "
             + "Gets the existing keytab without modification not effeting the prior keytab. The keytab in the response is base64 encoded.";
-    public static final String DELETE_SERVICE_PRINCIPAL_NOTES = "Deletes the pricipal";
-    public static final String DELETE_HOST_NOTES = "Deletes the host and all the principals associated with the host";
+    public static final String DELETE_SERVICE_PRINCIPAL_NOTES = "Deletes the principal.";
```

Comment:
Should this also state that it deletes any vault entries for the service principal that were created with the same cluster ID?